src/test/suite/commands/wizard/lwcGenerationCommand.test.ts

```diff
@@ -94,43 +96,57 @@ suite('LWC Generation Command Test Suite', () => {
     });
 
     test('Should return error status for landing page with invalid json', async () => {
+        const dirManager = await TempProjectDirManager.createTempProjectDir();
         const getWorkspaceDirStub = sinon.stub(
             WorkspaceUtils,
             'getStaticResourcesDir'
         );
-        getWorkspaceDirStub.returns(Promise.resolve('.'));
```

Comment:
This approach was breaking for me locally. Always dicey to use `.` as a working directory, as it's hard to know where the code is executed from. Moved to creating a temporary directory for this I/O.